src/strategies/aavegotchi-agip/index.ts

```diff
@@ -0,0 +1,126 @@
+import { formatUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+import { subgraphRequest } from '../../utils';
+export const author = 'Pixelcraft Studios';
```

Comment:
Hi.. Please change this to your GitHub or Pixelcraft Studios Github

Reply:
Hi! Have made this change